resources/definitions/ideagen3D_sapphire_plus.def.json

```
@@ -1,5 +1,5 @@
 {
-    "version": 2,
+    "version": 3,
```

Although @Ghostkeeper's comment was originally resolved, the version is now again changed to 3. Please change it back to 2 and the tests won't fail.


```suggestion
    "version": 2,
```